tests/Aws/Tests/Common/Client/ClientBuilderTest.php

```diff
@@ -396,6 +397,28 @@ public function testAllowsBackoffDisabling()
         $this->assertFalse($client->getConfig('client.backoff'));
     }
 
+    public function testAllowsBackoffMaxRetries()
+    {
+        $config = array(
+            'service' => 'dynamodb',
+            'region'  => 'us-east-1',
+            'service.description' => array(
+                'signatureVersion' => 'v2',
+                'regions' => array('us-east-1' => array('https' => true, 'hostname' => 'foo.com'))
+            ),
+            'client.backoff.retries' => 6
+        );
+
+        $client = ClientBuilder::factory('Aws\\DynamoDb')->setConfig($config)->build();
+        $this->assertEquals(6, $client->getConfig(Options::BACKOFF_RETRIES));
```

Comment:
Actually I use the  constant by looking at other tests, and in a way it made sense as you are also testing that the constant matches the actual string parameter, don't you think?